rh/admin.py

```diff
@@ -0,0 +1,3 @@
+# from django.contrib import admin
+
+# Register your models here.
```

Comment:
Actually can you just remove this file from the repository, since it has no code?